test/integration/api-resolver-query-writeable/server.js

```diff
@@ -0,0 +1,25 @@
+const next = require('next')
+const express = require('express')
+const { parse } = require('url')
+
+const dev = process.env.NODE_ENV !== 'production'
+const dir = __dirname
+const port = process.env.PORT || 3000
+
+const app = next({ dev, dir })
+const handleNextRequests = app.getRequestHandler()
+
+app.prepare().then(() => {
+  const server = express()
+  server.all('/:splat(*)', (req, res) => {
```

Comment:
This is not a valid route in Express v5

```suggestion
  server.all('/{*splat}', (req, res) => {
```

https://expressjs.com/en/guide/migrating-5.html#path-syntax